### PR TITLE
Add link to docs in `README`

### PR DESCRIPTION
**Pull request by navidcy**
@tomchor this is a placeholder; feel free to edit as per your liking.

### CODE REVIEW

**Comment by tomchor** [comment]
We should register a new version after this, so that there's another version of docs other than `dev`.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Stable documentation](https://img.shields.io/badge/documentation-stable%20release-blue?style=flat-square)](https://tomchor.github.io/Oceanostics.jl/stable/)
 
 
-Oceanostics is a Julia package created to facilitate getting diagnostics quantities in
+Oceanostics is a Julia package created to facilitate the creation of diagnosticquantities in
```
Comment (navidcy) [inline README.md:7]:
```suggestion
Oceanostics is a Julia package created to facilitate the creation of diagnostic quantities in
```

File: docs/src/index.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Markdown.parse_file(joinpath(@__DIR__, "..", "..", "README.md"))
 ```
 
 !!! note "This documentation is still under construction"
```
Comment (navidcy) [inline docs/src/index.md:6]:
```suggestion
!!! note "⚠️ Under construction! 🏗️"
```

**Review by tomchor — APPROVED** [review]

**Comment by tomchor** [comment]
@navidcy feel free to merge if you don't have any more suggestions

**Comment by tomchor** [comment]
Hmm, seems like the docs for the registered version aren't deploying. Only the docs for `dev` appear online. Something must be missing from CI config?

**Comment by navidcy** [comment]
> Hmm, seems like the docs for the registered version aren't deploying. Only the docs for `dev` appear online. Something must be missing from CI config?

Can you give me admin right on the repo to check the repo settings? Did you create the repo secret and public key for Documenter?

**Comment by tomchor** [comment]
I can't find an option to give you admin access. Maybe that feature is only available for enterprise users?

And no, I didn't create any deploy keys for this repo, but I think those aren't needed if you're deploying it from the `gh-pages` branch, no? At least that's what I understand when reading [this page](https://documenter.juliadocs.org/stable/man/hosting/#travis-ssh), although now that I read it again, the warning bubble [here](https://documenter.juliadocs.org/stable/man/hosting/#travis-ssh) implies that a documenter key is needed. Plus two of the github actions do have mentions to a documenter key:

https://github.com/tomchor/Oceanostics.jl/blob/13ad95a572ee6801217c76c977c2382d04d2250d/.github/workflows/documentation.yml#L27-L29

https://github.com/tomchor/Oceanostics.jl/blob/13ad95a572ee6801217c76c977c2382d04d2250d/.github/workflows/TagBot.yml#L13-L15

**Comment by tomchor** [comment]
OKay, I followed the instructions [here](https://documenter.juliadocs.org/stable/man/hosting/#travis-ssh) using as best as I could. Is there a way to test this without tagging a new version?

**Comment by navidcy** [comment]
Yeah, I'm pretty sure the tagbot needs it to trigger a docs built after a new release.

I manually created a release with a tag `v0.11.1-docs`. I didn't click the button to make this the latest release. I think that should trigger Documenter to built the docs for the v0.11.1 -- let's see.

**Comment by navidcy** [comment]
OK, the Documenter was triggered and pushed something in the `gh-pages` but then it does not appear in the docs website....

**Comment by tomchor** [comment]
Yeah, I see the files there in `gh-pages`. Weird, I'll investigate.

**Pull request merged**